snap/snapcraft.yaml

```diff
@@ -0,0 +1,67 @@
+name: graphs # you probably want to 'snapcraft register <name>'
+base: core22 # the base snap is the execution environment for this snap
+adopt-info: graphs
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+compression: lzo
+license: GPL-3.0
+architectures:
+  - build-on: amd64
+  - build-on: arm64
+
+slots:
+  graphs:
+    interface: dbus
+    bus: session
+    name: se.sjoerd.Graphs
+
+parts:
+  graphs:
+    # See 'snapcraft plugins'
+    plugin: meson
+    source: https://github.com/Sjoerd1993/Graphs.git
+    #source-tag: 'v1.5.2'
+    meson-parameters:
+      - --prefix=/snap/graphs/current/usr
+      - --buildtype=release
+    build-packages:
+      - blueprint-compiler
+    override-pull: |
+      craftctl default
+      patch -p1 < $CRAFT_PROJECT_DIR/snap/graphs.patch
+    override-build: |
+      craftctl default
+      pip install --prefix=$CRAFT_PART_INSTALL/usr matplotlib==3.7.1
+      pip install --target=$CRAFT_PART_INSTALL/usr/lib/python3/dist-packages scipy==1.10.1
+      sed -e '1c#!/usr/bin/env python3' -i ${CRAFT_PART_INSTALL}/snap/graphs/current/usr/bin/graphs
```

Comment:
In theory this file should have the correct shebang already (if meson picks up python correctly), is it necessary for this to be set again?

Reply:
Ah, I guess the interpreter isn't the same between buildtime and runtime then, that makes sense

Reply:
Yeah, in build it's gnome-42-2204-sdk, but in runtime its only gnome-42-2204